lsp-test/test/Test.hs

```diff
@@ -22,8 +22,8 @@ import           Control.Monad
 import           Control.Lens hiding (List, Iso)
 import           Language.LSP.Test
 import           Language.LSP.Protocol.Message
-import           Language.LSP.Protocol.Types hiding (message, applyEdit)
-import qualified Language.LSP.Protocol.Types as LSP
+import           Language.LSP.Protocol.Types
+import qualified Language.LSP.Protocol.Lens as J
```

Comment:
This use of `J` is very weird. I _believe_ it dates back to when it was really just the JSON types being imported. Or at least I can't think why else it was like that. I've been trying to slowly get rid of it in favour of qualifying as LSP (although that is longer...).

Reply:
Would L be better? It's nice for lens stuff to just use one character because the getters tend to be pretty short too. LSP is longer and then is also used already by all the non lens stuff when it's imported qualified.

Reply:
Sure, L seems fine.